fees/blazingbot.ts

```diff
@@ -90,11 +90,9 @@ const adapter: SimpleAdapter = {
     },
     [CHAIN.SOLANA]: {
       fetch: fethcFeesSolana,
-      runAtCurrTime: true,
       start: '2024-11-23',
     },
   },
-  isExpensiveAdapter: true,
```

Comment:
if you use dune query it require set `isExpensiveAdapter` true it can't remove

Reply:
But besides this the runAtCurrTime removal will get us the historic data we want?